udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,14 +37,44 @@ private String arrayListToJson(ArrayList<String> inputList) {
     return json;
   }
 
+  /**
+   * Obtains comments json to display upon request.
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    comments = new ArrayList<String>();
-    comments.add("What's good");
-    comments.add("Howdy friend!");
-    comments.add("Hello homie.");
     String outputJson = arrayListToJson(comments);
     response.getWriter().println(outputJson);
   }
+
+  /**
+   * Writes comment data to ArrayList data
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getRequestParameter(request, "text-input", "");
+    boolean toErase = Boolean.parseBoolean(getRequestParameter(request, "erase", "false"));
+
+    if (toErase) {
```

Comment:
`shouldErase` may sound more fluent. Compare: `if to erase` and `if should erase` when reading this condition.